apps/app/src/components/templates/layout.tsx

```diff
@@ -19,7 +19,7 @@ const Layout = () => {
       {isBrowser ? (
         <div className="flex w-screen h-screen bg-current">
           <div className="flex m-auto flex-col">
-            <img src="/logo-light.png" alt="logo" className="h-32 mx-auto my-8" />
+            <img src="/logo-light.svg" alt="logo" className="h-32 mx-auto my-8" />
```

Comment:
Can you make this change depending on light theme or dark theme ?

Reply:
useTheme() should be available here

Reply:
(Providers has the theme provider)

Reply:
sure

Reply:
Done